Projects/IntelligentObjectAvoider/README.md

```diff
@@ -10,9 +10,11 @@ The `servo` is particularly useful, because it gives the distance sensor a 120 d
 
 In order to test this script, it's advised to create at your place a small labyrinth where the `GoPiGo3` can be put in. Books, bags, stools can be used. Avoid creating very sharp corners in your little maze or too small "corridors" as your `GoPiGo3` can get stuck.
 
-This script can only run with `Python 3`.
-Don't try to run it with `Python 2` or otherwise, it will fail. Run it with the following command:
-* `python3 robot.py`
+You need to do the following things in order to be able to launch the `robot.py` script:
+1. Install `scikit-learn` package. A `pip install scikit-learn` should be enough (be mindful of whether you need to use `sudo` or not).
```

Comment:
Figure what? How to install a package? If that's so, then they should also learn how to use a terminal, but this doesn't mean we have to do it - this can go really far and we can lose ourselves into details.

How do you see it being written?

Reply:
I meant for **(be mindful of whether you need to use `sudo` or not)** 

How about : 
*Use `pip3 install scikit-learn`  *
*It is possible that you will need to use `sudo pip3 install scikit-learn` if the previous command fails*

Also, yeah, on Raspbian, you do need to use sudo